check_process

```diff
@@ -12,7 +12,6 @@
 		setup_private=0
 		setup_public=0
 		upgrade=1
-		upgrade=1	from_commit=8d0acb6953a4c4c2206985c7d9881c5b7b637ebd
```

Comment:
Why do you remove this test ?

Reply:
Because it was an upgrade to an older version... I don't think it can work...

Reply:
An upgrade __from__ an older version. That's the point of this test.
If your upgrade script doesn't work when upgrading from an older version, then it has to be fixed.

Reply:
OK then, misunderstanding of that functionality, So I haven't tested ^^
I roll back the check_process